### ISSUE TITLE
Commit 71f0593c342 breaks landing gear model

### ISSUE THREAD

**Issue by pgkirsch** [hoburg/gpkit#889]
The latest version of landing gear (including before and after @bqpd's most recent improvements) is returning the following error after commit 71f0593c342:

```python
Beginning signomial solve.
ERROR: An unexpected error occurred while tokenizing input
The following traceback may be corrupted or invalid
The error message is: ('EOF in multi-line string', (28, 50))

---------------------------------------------------------------------------
RuntimeWarning                            Traceback (most recent call last)
/Users/philippekirschen/Documents/MIT/Research/GPkit/gpkit/gpkit/models/gpkitmodels/aircraft/landing_gear.py in <module>()
    326 
    327 if __name__ == "__main__":
--> 328     LandingGear.test()

/Users/philippekirschen/Documents/MIT/Research/GPkit/gpkit/gpkit/models/gpkitmodels/aircraft/landing_gear.py in test(cls)
    322         """Tests the standalone landing gear model"""
    323         m = cls.standalone737()
--> 324         sol = m.localsolve()
    325         print sol.table()
    326 

/Users/philippekirschen/Documents/MIT/Research/GPkit/gpkit/gpkit/constraints/prog_factories.pyc in solvefn(self, solver, verbosity, skipsweepfailures, *args, **kwargs)
     79         else:
     80             self.program, solvefn = genfunction(self, verbosity-1)
---> 81             result = solvefn(solver, verbosity, *args, **kwargs)
     82             solution.append(result)
     83         solution.program = self.program

/Users/philippekirschen/Documents/MIT/Research/GPkit/gpkit/gpkit/constraints/signomial_program.pyc in localsolve(self, solver, verbosity, x0, rel_tol, iteration_limit, **kwargs)
    102     The last result is available in Model.program.gps[-1].result. If the gps
    103     appear to be converging, you may wish to increase the iteration limit by
--> 104     calling .localsolve(..., iteration_limit=NEWLIMIT).""" % len(self.gps))
    105             gp = self.gp(x0, verbosity=verbosity-1)
    106             self.gps.append(gp)  # NOTE: SIDE EFFECTS

RuntimeWarning: problem unsolved after 52 iterations.

    The last result is available in Model.program.gps[-1].result. If the gps
    appear to be converging, you may wish to increase the iteration limit by
    calling .localsolve(..., iteration_limit=NEWLIMIT).
```

Strangely, the full aircraft models works ok.

**Comment by pgkirsch** [comment]
Wait it looks like @bqpd already caught this in his comments on 71f0593. But you said you pushed a fix to master, but that fix didn't resolve it?

**Comment by bqpd** [comment]
It got broken again by https://github.com/hoburg/gpkit/commit/571e9cccd4654126a970ddfc8ca3d33b58b4e947 :-/

**Comment by bqpd** [comment]
Fixed by #884
